### PR TITLE
Add translation for zh-TW

### PR DESCRIPTION
**Pull request by et169tkm**
Hi,

Thanks for your great work, your emulator is fabulous!!! (And saved me 18 gold medals :P )

I've forked your repository just to add translation, I have no intention of making a duplicate.
I've added  the zh-TW translation in a few places.
Would you review my commits? And if you don't like my approach, please let me know and I'm happy to follow the approach you like.

Many Thanks,
Eric

### CODE REVIEW

**Comment by WindWT** [comment]
What you need is dump & convert files from TW/WeGame server(`/data/*.json`), not translate data from CN/XinDong server.
Some of those data extract tools are not open-sourced(but it's pretty easy to reinvent the wheel), others are other people's open-source project(since it's a unity game, you know how to choose tools). If you just want an rune simulator for TW/WeGame server, make your own extract tool, extract data, and replace `/data` might be an better idea.
Supporting mutiple servers' data is another approach, but because of ~~I have new waifu~~ I'm losing interest on this game and some other reasons(mostly health issues), I'm lazy adding functions for this project. If you can't wait, you can just publish your translated version without my approvement.



TLDR:
1. **NO**, I won't merge this.
2. **YES**, you can do **anything** you like with this code.
3. I may or may not add function for switching different server datas.

**Comment by WindWT** [comment]
I didn't check mails for a long time, sorry to keep you waiting.

**Comment by et169tkm** [comment]
Thanks for your reply. I hope you are well!
Thanks for the suggestion, I thought translation is what you would do since there are some translations in your code, I wasn't aware the data on different servers are different.It's a great project that you've put together, thanks for being so generous to opensource your project!
 FYI my local copy works fine for me, I don't intend to publish duplicate.

Again, I hope you are well :)
Many thanks
Eric

    On Wednesday, 4 April 2018, 14:02:41 GMT+1, WindWT <notifications@github.com> wrote:  
 
 
What you need is dump & convert files from TW/WeGame server(/data/*.json), not translate data from CN/XinDong server.
Some of those data extract tools are not open-sourced(but it's pretty easy to reinvent the wheel), others are other people's open-source project(since it's a unity game, you know how to choose tools). If you just want an rune simulator for TW/WeGame server, make your own extract tool, extract data, and replace /data might be an better idea.
Supporting mutiple servers' data is another approach, but because of  I'm losing interest on this game and some other reasons(mostly health issues), I'm lazy adding functions for this project. If you can't wait, you can just publish your translated version without my approvement.

TLDR:
   
   - NO, I won't merge this.
   - YES, you can do anything you like with this code.
   - I may or may not add function for switching different server datas.
